pkg/manager/member/orphan_pods_cleaner.go

```diff
@@ -90,8 +91,10 @@ func (opc *orphanPodsCleaner) Clean(tc *v1alpha1.TidbCluster) (map[string]string
 
 		err = opc.podControl.DeletePod(tc, pod)
 		if err != nil {
+			glog.Errorf("orphan pods cleaner: failed to clean orphan pod: %s/%s, %v", ns, podName, err)
```

Comment:
I generally prefer to annotate errors: that way there is a single error data structure with all the information that can be logged in one spot, and if there is an API request sent back in the response.

Reply:
Yes, our error logs style need a lot of improvements. We may discuss it in a new issue?

This pr tries to add detailed logs when something changes to help us diagnose problems.

Reply:
It is up to you. I think either way you should get a detailed log to help diagnose problems. If there is concurrent logging annotation will be simpler to understand.

Reply:
what is "logging annotation"?